RELEASE_NOTES.md

```diff
@@ -2,6 +2,13 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 0.26.3 (March 5, 2024)
+
+🧑‍🔬 Improved analytics to help us better diagnose problems with patches.
```

Comment:
```suggestion
🧑‍🔬 Show in the console if patches were built with --force or native code changes.
```

Reply:
I would phrase it in terms of the user benefit. My sense is the general public is more skeptical of "analytics" these days. :)

Reply:
Updated per offline convo